core/base/inc/TPluginManager.h

```diff
@@ -155,7 +155,7 @@ friend class TPluginManager;
       // resource ... and both SetParams and Execute ends up taking the lock
       // individually anyway ...
 
-      R__LOCKGUARD2(gInterpreterMutex);
+      R__LOCKGUARD(gInterpreterMutex);
```

Comment:
It depends on whether the mutex guard is guaranteed to be initialized or not ....